### PR TITLE
Update cdn

### PR DESCRIPTION
**Pull request by smalls**
In progress; as the last commit notes, this isn't working yet.

### CODE REVIEW

File: dev/artifacts/Products/Products.recipes [hunk 1]
```diff
@@ -59,17 +58,17 @@ recipe
   GiftList
     person <- view1
   ProductMultiplexer
-    products <- view2
+    list <- view2
```
Comment (smalls) [inline dev/artifacts/Products/Products.recipes:61]: I should have mentioned this, I'm not sure which is preferred (`products` or `list`) but in this change I was thinking that it was easier to keep arcs & arcs-cdn in sync if they used the same identifier.

`products` is more descriptive, though. Should I change this and the version in arcs to that?

Comment (sjmiles) [reply dev/artifacts/Products/Products.recipes:61]: Fwiw, my hope is that soonish the Multplexers are generic, in which case the input parameter will need a generic name, like `list`.

Comment (smalls) [reply dev/artifacts/Products/Products.recipes:61]: Oh yes! Thanks makes total sense. And there's the start to that in ProductMultiplexer2 (one of the parameters is generic).

File: dev/artifacts/Products/ProductMultiplexer.manifest [hunk 2]
```diff
@@ -1,19 +1,31 @@
...
+  description `${hostedParticle} for ${list}`
+
+// Same as ProductMultiplexer above, but with an additional connection.
+shape HostedParticleShape
+  HostedParticleShape(in ~a, in [~a])
+  consume annotation
+
+// TODO: rename to a generic (non product specific) multiplexer.
+particle ProductMultiplexer2 in 'source/ProductMultiplexer.js'
+  ProductMultiplexer2(host HostedParticleShape hostedParticle, in [~a] list, in [Product] others)
```
Comment (mariakleiner) [inline dev/artifacts/Products/ProductMultiplexer.manifest]: Please, change others to [~a] too

**Review by mariakleiner — APPROVED** [review]

**Pull request merged**